README.md

```diff
@@ -40,4 +44,4 @@ Special thanks to:
 
 If you want to support this addon you can donate to the following addresses:
  * bitcoin: 159Y9BLcfHyrp6wj6f3syEuk92xkRVTiie
- * monero: 482QYZaagALWtPmwbptwBaexDYmcVsJrhJp2VVjTgjYA3Kk1YyMdSg9Wz2qz1Gh31E843PFVCDWS4hR4Bjf6ipWuB9iz2cs
```

Comment:
Why did you change something here?

Reply:
No idea, i just copied and pasted it from hear.
https://github.com/pyllyukko/user.js#fingerprinting-tests

Did i move that, or add it, i am not sure.